apps/frontend/src/libs/components/toast-container/styles.module.css

```diff
@@ -0,0 +1,14 @@
+.toast-container {
+	--toastify-color-light: #ffffff;
+	--toastify-color-dark: var(--color-background-primary);
+	--toastify-color-info: var(--color-brand-primary);
+	--toastify-color-success: var(--color-success);
+	--toastify-color-warning: var(--color-warning);
+	--toastify-color-error: var(--color-danger);
+	--toastify-toast-width: 320px;
+	--toastify-toast-min-height: 64px;
+	--toastify-toast-max-height: 800px;
+	--toastify-font-family: sans-serif; /* TODO: change font family */
+	--toastify-text-color-light: #757575;
+	--toastify-text-color-dark: #ffffff;
```

Comment:
Do we need both dark and light? and can't we use common variables for this? and the same question for `--toastify-color-light`, `--toastify-color-dark`

Reply:
Yeah, you right. Let's use default values for this (or what do you mean by "common variables"). If so, then I will just delete those lines

Reply:
Oh, should I also delete `--toastify-toast-width`, `--toastify-toast-min-height` and `--toastify-toast-max-height`? (I use default values for those, but keep them, if we will need to adjust the values)